packages/terra-time-input/package.json

```
@@ -30,7 +30,7 @@
     "classnames": "^2.2.5",
     "keycode-js": "^2.0.1",
     "prop-types": "^15.5.8",
-    "terra-button-group": "^3.0.0",
+    "terra-button-group": "3.37.0",
```

I'm going to try revert to see what the failures are

```suggestion
    "terra-button-group": "^3.0.0",
```